### PR TITLE
[16.0] [MIG] website_sale_product_minimal_price

### PR DESCRIPTION
**Pull request by sbiosca-s73**
Migration module website_sale_product_minimal_price to v16.0

### CODE REVIEW

File: website_sale_product_minimal_price/models/product_template.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def _get_cheapest_info(self, pricelist):
             for qty in [1, 99999999]:
                 product_price = product.with_context(
                     quantity=qty, pricelist=pricelist.id
-                ).price
+                ).lst_price
```
Comment (chienandalu) [inline website_sale_product_minimal_price/models/product_template.py]: `_get_product_price` probably fits better the needed migration value: https://github.com/odoo/odoo/blob/4d119ea9fafd25ed4709eb868132b28a54a41bc0/addons/product/models/product_pricelist.py#L76

Check the migration commit for this field: https://github.com/odoo/odoo/commit/9e99a9df464d97a74ca320d200599f7dce2d3050

File: website_sale_product_minimal_price/controllers/main.py [hunk 2]
```diff
@@ -76,9 +76,9 @@ def get_combination_info_pricelist_atributes(self, product_id, **kwargs):
         )
         qty_list = sorted(set(qty_list.mapped("min_quantity")))
         res = []
-        last_price = product.with_context(quantity=0).price
+        last_price = product.with_context(quantity=0).lst_price
```
Comment (chienandalu) [inline website_sale_product_minimal_price/controllers/main.py]: The same here

File: website_sale_product_minimal_price/controllers/main.py [hunk 3]
```diff
@@ -76,9 +76,9 @@ def get_combination_info_pricelist_atributes(self, product_id, **kwargs):
...
         for min_qty in qty_list:
-            new_price = product.with_context(quantity=min_qty).price
+            new_price = product.with_context(quantity=min_qty).lst_price
```
Comment (chienandalu) [inline website_sale_product_minimal_price/controllers/main.py]: And here

**Review by chienandalu — CHANGES_REQUESTED** [review]
Thanks @sbiosca-s73 Some comments: below

File: website_sale_product_minimal_price/models/product_template.py (hunk 1 above)
Comment (sbiosca-s73) [reply website_sale_product_minimal_price/models/product_template.py]: Okey, thanks

**Comment by sbiosca-s73** [comment]
@chienandalu ping

**Review by chienandalu — COMMENTED** [review]
Tests seems to be failing

**Comment by sbiosca-s73** [comment]
@chienandalu ping

File: website_sale_product_minimal_price/readme/CONTRIBUTORS.rst
Comment (chienandalu) [inline website_sale_product_minimal_price/readme/CONTRIBUTORS.rst:1]: nitpick: Add yourself to the contributors list

**Review by chienandalu — APPROVED** [review]
Thanks! @sbiosca-s73 


please review @stefan-tecnativa

**Comment by stefan-tecnativa** [comment]
Tested on runbot, I will explain my test and the results I obtained.
I selected the "Conference Chair" product, which has an attribute "Legs" with two values: steel and aluminium. I've assigned an `price_extra` to the steel value of two dollars. Then i went to the shop and I correctly saw under the chairs icon the "From 35$" text, which is the correct calculation of the minimum price as the chais is 33$. 
Then i went back and I changed the steel `price_extra` to 8$. When i go back to the shop the text is again correctly displayed but when I open the product the information is not correct. The price "label" is not correctly updated, but the final price calculation is correct. 
imagen

**Comment by sbiosca-s73** [comment]
@chienandalu please apply the "no stale" label. I'm going to review the comment of @stefan-tecnativa.

Thanks!

**Comment by sbiosca-s73** [comment]
@chienandalu @stefan-tecnativa I have conducted different tests to verify what Stefan mentioned. What he says is correct, but this is unrelated to the module being migrated in this PR. 

Regarding the issue he mentioned, I have found a solution. Every time the `price_extra` of a value is changed, the `website_sale` module should be updated to reflect the changes on the website; otherwise, we encounter the issue Stefan mentioned. 

However, as mentioned before, I don't see the module `website_sale_product_minimal_price` affecting the situation at any point.

Thanks!

**Review by chienandalu — APPROVED** [review]
Thanks @sbiosca-s73 :+1: 

@pilarvargas-tecnativa can you review it as well?

File: website_sale_product_minimal_price/readme/CONFIGURE.rst [hunk 4]
```diff
@@ -0,0 +1,2 @@
+#. Go to *Website > Configuration > Settings > Pricing > Pricelist* and active the option *Multiple
```
Comment (pilarvargas-tecnativa) [inline website_sale_product_minimal_price/readme/CONFIGURE.rst]:
```suggestion
#. Go to *Website > Configuration > Settings > Shop - Products* and active the option *Multiple
```

**Comment by pilarvargas-tecnativa** [comment]
LGTM, please make this minor modification to the USAGE.rst file.

**Comment by sbiosca-s73** [comment]
@pilarvargas-tecnativa changes done

**Review by pilarvargas-tecnativa — APPROVED** [review]
:+1:  thanks!

**Pull request merged**